src/components/app/AppViewRouter.js

```diff
@@ -111,7 +114,12 @@ class AppViewRouterImpl extends PureComponent<AppViewRouterProps> {
         // should be 'ROUTE_NOT_FOUND' or 'PROFILE_LOADED'.
         (phase: 'ROUTE_NOT_FOUND');
         return (
-          <Home specialMessage="The URL you came in on was not recognized." />
+          <Localized
+            id="AppViewRouter--home-special-message"
```

Comment:
I think this id isn't well written because it doesn't convey the right meaning. This is used when the `phase` is `ROUTE_NOT_FOUND`, or when it's not known. Therefore the ID should be something like `AppViewRouter--route-not-found--home`.